impl/src/main/java/io/kroxylicious/testing/kafka/clients/CloseableAdmin.java

```diff
@@ -292,19 +298,6 @@ public DescribeConfigsResult describeConfigs(Collection<ConfigResource> resource
         return instance.describeConfigs(resources, options);
     }
 
-    @SuppressWarnings("deprecation")
-    @Override
-    @Deprecated
-    public AlterConfigsResult alterConfigs(Map<ConfigResource, Config> configs) {
-        return instance.alterConfigs(configs);
```

Comment:
For backward compatibility sake we ought to maintain the method and use reflection to try to invoke the method on instance.  That way, we keep runtime compatibility for users with an older version of Kafka on the classpath who are using the deprecated methods.  My hunch is that the number of users doing this will be zero so I've going to avoid doing the work (& writing the tests).